testdata/get-configuration.json

```diff
@@ -12,9 +14,8 @@
     "orderflow_proxy": {
         "flashbots_orderflow_signing_address": "0x00",
         "builder_endpoint": "http://127.0.0.1:8645",
-        "local_listen_addr": "0.0.0.0:443",
+        "local_listen_addr": "127.0.0.1:3443",
         "public_listen_addr": "0.0.0.0:5544",
-        "cert_hosts": "1.2.3.4,localhost,127.0.0.1",
```

Comment:
what's the rationale for replacing `cert_hosts` with a fixed IP and DNS entry? should be fine anyway, just curious

Reply:
It is easier to compose a string with CN names for the certificate from the individual components like IP address and DNS name when needed. Top level IP address and DNS name may come in handy for some other services (though we don't have a use case right now).